src/main/resources/com/google/api/codegen/py/method_sample.snip

```diff
@@ -1,5 +1,5 @@
 @extends "py/common.snip"
-@extends "py/init.snip"
+@extends "py/simple_init.snip"
```

Comment:
uhm, s/`simple`/`sample`/ here and in the filename

Reply:
Done